scripts/sync-up-helm-chart.sh

```diff
@@ -47,17 +47,18 @@ fi
 rm -f ./config/crds/*.yaml
 
 # Generate CRD manifest files
-command -v controller-gen &> /dev/null || (cd ${ROOT_DIR}/tools && go install sigs.k8s.io/controller-tools/cmd/controller-gen)
-controller-gen crd:trivialVersions=true paths="./pkg/apis/..." output:crd:artifacts:config=config/crds
+(cd ${ROOT_DIR}/tools && GOBIN=${ROOT_DIR}/bin go install sigs.k8s.io/controller-tools/cmd/controller-gen)
```

Comment:
so a dumb question, how do we ensure the version? 🤔

Reply:
Using a separate tools module (https://github.com/kubernetes-sigs/kubefed/blob/master/tools/go.mod#L7).